Header header


geometry_msgs/Vector3 baseVector
float32 minAlpha
float32 maxAlpha
float32 minBeta
float32 maxBeta
float32 minGamma
float32 maxGamma
float32 minKappa
float32 maxKappa
